src/mixins/legal-api-mixin.ts

```diff
@@ -11,39 +11,16 @@ import { ActionBindingIF, CorrectionFilingIF, IncorporationFilingIF } from '@/in
 import { FilingTemplateMixin } from '@/mixins'
 
 /**
- * Mixin that provides the integration with the legal api.
+ * Mixin that provides integration with the Legal API.
  */
 @Component({})
 export default class LegalApiMixin extends Mixins(FilingTemplateMixin) {
-  readonly NAME_REQUEST = 'nameRequest'
   readonly INCORPORATION_APPLICATION = 'incorporationApplication'
 
   // Global getters
   @Getter getFilingId!: number
   @Getter getBusinessId!: string
 
-  // Global setters
-  @Action setFilingId!: ActionBindingIF
```

Comment:
This line of code was weird... It's not used in this file, but it was used in Correction.vue and Alteration.vue (which imported this mixin). I didn't know that was possible.

Anyway, I removed it from here and added it to the views where they are used, for clarity.